provisioning/cloudformation/telemetry-regression-alerts.json

```diff
@@ -118,7 +118,7 @@
           "    -o DPkg::Options::=--force-confdef \\",
           "    -o DPkg::Options::=--force-confold \\",
           "    install git nodejs npm python-numpy python-opencv python-matplotlib python-pip",
-          "pip install awscli boto simplejson",
+          "pip install awscli boto simplejson beautifulsoup4",
```

Comment:
Changes to this file appear unrelated to the "termination time" change - please send this change as a separate PR